src/core/settings.py

```diff
@@ -133,3 +134,7 @@
         'rest_framework_simplejwt.authentication.JWTAuthentication',
     ),
 }
+
+SIMPLE_JWT = {
+    'ACCESS_TOKEN_LIFETIME': timedelta(days=100),
```

Comment:
LIfe time of token should be small number, 1 min - few days. This is depends on type of web app